### DISCUSSION TITLE
PedalinoMini custom design

### DISCUSSION THREAD

**Discussion by alf45tar** [alf45tar/PedalinoMini#337] [General]
@Poznic 
Please add here your spec for PedalinoMini custom design.

**Comment by Poznic** [comment]
Hello,

Thanks for your message and my respect to your work. What you achieved so far looks really great. I can't wait to build and use one.
I'll start by making the original size of course.

For a custom design I'd be looking for a shorter version with these specs : 
- 4 buttons
- Screen placed in the middle (if it is possible)
- One of the expression input can be removed as I only use one expression pedal in my setup.

Something like the one built by user teopost shown in the "End User Built" section but with 4 buttons. 

I hope my request is doable for you. 

Thanks in advance.

**Reply by alf45tar** [reply]
TTGO T-Display has the screen attached to board. Putting screen in the middle you cannot connect the USB connector.

**Reply by Poznic** [reply]
Yes that's I was thinking after I posted the request :-). Just a 4 button version would be great then.

**Comment by alf45tar** [comment]
Dimension are 150 x 100 x 26 mm

@Poznic feel free to comment. Do you have a better idea to place the components?


![Body4Top](https://user-images.githubusercontent.com/35426671/168419396-7393b28d-d0fe-46d6-9dac-30a002c36fbb.png)
![Body4Bottom](https://user-images.githubusercontent.com/35426671/168419399-65f7b542-9ec8-49ea-8fbe-387fd3f0b617.png)
![Base4](https://user-images.githubusercontent.com/35426671/168419404-3f8238ec-f415-473c-a4cf-e8ba882b7d06.png)

**Reply by Poznic** [reply]
That looks pretty good. I think it is fine like it is. Thanks a lot for your work.

**Reply by alf45tar** [reply]
How can I send the STL files to you?

**Reply by Poznic** [reply]
You can put them in my Google drive : https://drive.google.com/drive/folders/1078bMAnaqd9U06Jdbdg4vwLfZKjlHkFN

**Reply by alf45tar** [reply]
Files uploaded

**Reply by Poznic** [reply]
I can see them. Well received. Thanks. Have à nice week-end.
